jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/lambda/go/GoSamProjectWizard.kt

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.go
+
+import com.goide.sdk.GoSdkService
+import com.goide.sdk.combobox.GoSdkChooserCombo
+import com.goide.vgo.configuration.VgoProjectSettings
+import com.intellij.facet.ui.ValidationResult
+import com.intellij.openapi.progress.ProgressIndicator
+import com.intellij.openapi.roots.ModifiableRootModel
+import com.intellij.openapi.ui.TextFieldWithBrowseButton
+import com.intellij.openapi.ui.ValidationInfo
+import com.intellij.openapi.vfs.VirtualFile
+import software.amazon.awssdk.services.lambda.model.PackageType
+import software.amazon.awssdk.services.lambda.model.Runtime
+import software.aws.toolkits.jetbrains.services.lambda.wizard.SamAppTemplateBased
+import software.aws.toolkits.jetbrains.services.lambda.wizard.SamNewProjectSettings
+import software.aws.toolkits.jetbrains.services.lambda.wizard.SamProjectTemplate
+import software.aws.toolkits.jetbrains.services.lambda.wizard.SamProjectWizard
+import software.aws.toolkits.jetbrains.services.lambda.wizard.SdkSelector
+import software.aws.toolkits.jetbrains.utils.ui.validationInfo
+import software.aws.toolkits.resources.message
+import javax.swing.JComponent
+import javax.swing.JLabel
+
+class GoSamProjectWizard : SamProjectWizard {
+    override fun createSdkSelectionPanel(projectLocation: TextFieldWithBrowseButton?): SdkSelector = GoSdkSelectionPanel()
+
+    override fun listTemplates(): Collection<SamProjectTemplate> = listOf(
+        SamHelloWorldGo()
+    )
+}
+
+class GoSdkSelectionPanel : SdkSelector {
+    private val interpreterPanel = GoSdkChooserCombo()
```

Comment:
Should be registered disposable

Reply:
With what as the parent disposable? None of the other sdk selection panels register it as a disposable?

Reply:
Ya, not sure looking back at the code what we would use. I dont see us having access to one easily... Oh well.